vault/diagnose/tls_verification.go

```diff
@@ -28,64 +37,119 @@ func ListenerChecks(listeners []listenerutil.Listener) error {
 		}
 		_, ok := tlsutil.TLSLookup[l.TLSMinVersion]
 		if !ok {
-			return fmt.Errorf(minVersionError, l.TLSMinVersion)
+			err := fmt.Errorf("listener at address: %s has error %s: ", listenerID, fmt.Sprintf(minVersionError, l.TLSMinVersion))
```

Comment:
is it valuable to wrap the underlying error here (rather than insert a string)? like would that ever feasibly benefit calling code or tests?

Reply:
There isn't an underlying error constant here -- `minVersionError` is a string.

Reply:
yes, but it was previously an error. i'm wondering if we'd like it to still be an error, and be wrapped?

Reply:
I'm not sure I follow -- `minVersionError` was always a string as far as I'm aware.

Reply:
we discussed in slack, and it sounds like doing so won't be useful. resolving!